sources/reference/tutorials/batch.md

```diff
@@ -0,0 +1,608 @@
+# Running Chaos Toolkit experiments as AWS Batch Jobs
+
+It is common when using AWS for hosting your infrastructure that you'll have
+strict security policies in place. These policies will usually only allow for
+internal traffic within AWS, amongst various other things. A question we're
+asked a lot is `can I run Chaos Toolkit from AWS, to run against AWS?`.
+The answer is simply, yes, you can.
+
+## Why Batch?
+
+You may have followed our [Running Chaos Toolkit from an EC2 instance][] guide
+and wonder why we would write a guide for Batch - Batch has some benefits
+over EC2:
+
+[Running Chaos Toolkit from an EC2 instance]: ../containerising
+
+* Your infrastructure isn't running all the time
+* You can use Docker images to encapsulate your experiment environment
+* You can submit multiple jobs to run different experiments rather than blocking
+on one experiment in an EC2 instance
+
+## Why not ECS and Fargate?
+
+We sometimes get asked about how to run Chaos Toolkit **on** ECS with Fargate,
+whilst we understand why you might want to do this, Chaos Toolkit experiments
+aren't analogous to something like a microservice. We don't run Chaos Toolkit
+continuously and request it to run jobs, rather we invoke Chaos Toolkit when we
+want to use it.
+
+Because of this difference in thinking, we recommend you use Batch (With Fargate
+as the compute provider) to invoke Chaos Toolkit experiments.
+
+## The Steps
+
+For the purposes of this guide, we'll run you through setting up your Chaos Toolkit
+experiments manually. If however, you're familiar with [the AWS Cloud Development
+Kit (CDK)][], we have an example repository deploying the same infrastructure
+using CDK [here][].
```

Comment:
Is https://github.com/chaostoolkit/chaostoolkit-issues-playground the correct link?

Reply:
Doh.